src/blocks/Footer/Newsletter.js

```diff
@@ -3,11 +3,13 @@ import styled from 'styled-components'
 import Button from 'components/Button'
 import BaseInput from 'components/Input'
 import Typography from 'components/Typography'
+import Card from 'components/Card'
 import get from 'utils/get'
 
-const Container = styled.div`
-  background-color: ${get('colors.valhalla.400')};
-  border-radius: 16px;
+const Container = styled(Card)`
```

Comment:
What does this do? Does `Container` inherit from the styles of `Card`?

Reply:
Yes absolutely! :)